tests/Behat/Context/Ui/Admin/ChannelPricingLogEntryContext.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Tests\Sylius\PriceHistoryPlugin\Behat\Context\Ui\Admin;
+
+use Behat\Behat\Context\Context;
+use Sylius\Component\Core\Model\ProductVariantInterface;
+use Webmozart\Assert\Assert;
+use Tests\Sylius\PriceHistoryPlugin\Behat\Page\Admin\PriceHistory\IndexPageInterface;
+
+final class ChannelPricingLogEntryContext implements Context
```

Comment:
```suggestion
final class BrowsingChannelPricingLogEntriesContext implements Context
```
but I'm not sure

Reply:
I think there won't be that many contexts with that view, so I decided to leave it as it is 🤔